workflow-templates/build-pr.properties.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do you have a feel for how this will work? When someone first creates repo, can they select multiple templates? Or will they have to select one and then copy others?

Reply:
I think you can import as many as you want, might just have to do it one at a time. (I think with the "New Workflow" button)